docs/release_notes/0.90.0.md

```diff
@@ -0,0 +1,18 @@
+# Release 0.90.0
+
+
+## Features
+
+- Added propagate asg tags to propagate labels and taints on creation (#4962)
```

Comment:
The linked issue's title is more helpful.
```suggestion
- Automatically create ASG tag from nodegroup's labels and taints (#4962)
```

Reply:
I really don't like editing release notes, we should make sure our PR titles are good enough so the release process becomes smooth for us. I'll make an exception here tho since the suggestion is nice 👌🏻